graphistry/arrow_uploader.py

```diff
@@ -548,7 +597,7 @@ def post_file(self, file_path, graph_type='edges', file_type='csv'):
                 f'{base_path}/api/v2/upload/datasets/{dataset_id}/{graph_type}/{file_type}',
                 verify=self.certificate_validation,
                 headers={'Authorization': f'Bearer {tok}'},
-                data=file.read()).json()
+                data={"org_name": self.org_name}, files={"upload_file": file.read()}).json()
```

Comment:
will this work? i'm not sure we're doing a multipart or whatever upload

more broadly, metadata is supposed to get set during dataset creation, or if a file part (vs formal File ) as part post_arrow, a url param (opts)

Reply:
This has been removed earlier